recipes/folly/all/conan_deps.cmake

```diff
@@ -0,0 +1,39 @@
+# Set the dependency flags expected by https://github.com/facebook/folly/blob/v2023.12.18.00/CMake/folly-deps.cmake
```

Comment:
I really like handling this here rather than in a lot of patches :D  And the link to that folly script is really useful